app/components/primer/alpha/overlay.html.erb

```diff
@@ -6,7 +6,20 @@
     <% if content.present? %>
       <%= content %>
     <% else %>
-      <%= body %>
+      <% if bodies.size == 1 %>
```

Comment:
I know visually it doesn't seem to matter, but can we handle the zero case here?